management/clipper_manager.py

```diff
@@ -793,8 +797,12 @@ def add_container(self, model_name, model_version):
             # Look up model info in Redis
             model_key = "{mn}:{mv}".format(mn=model_name, mv=model_version)
             result = local(
-                "redis-cli -h {host} -p 6379 -n {db} hgetall {key}".format(
-                    host=self.host, key=model_key, db=REDIS_MODEL_DB_NUM),
+                "redis-cli -h {host} -p {redis_port} -n {db} hgetall {key}".
+                format(
```

Comment:
The format order of `{db}` and `{key}` is swapped. We're setting `{db}` to `key` and `{key}` to db.